utils/db/repos.ts

```diff
@@ -170,3 +170,50 @@ export const getRepoConfigByUserAndRepo = async (provider: string, repoName: str
 	}
 	return userRows[0].config;
 }
+
+export const insertRepoConfig = async (userId: string, repoIds: number[]) => {
+	const query = `
+		INSERT INTO repo_config (repo_id, user_id, auto_assign, comment_setting)
+		SELECT id, $1, true, true
+		FROM unnest($2::INT[]) AS t(id)
+		ON CONFLICT DO NOTHING
```

Comment:
Might not need an ON CONFLICT clause if you DO NOTHING in it

Reply:
but If I don't use the on conflict then in case of conflict it shows error for repo_config table due to its fk coinstraint